### PR TITLE
New witches

### PR DESCRIPTION
**Pull request by interfector**
Here's some new witches: Maldea, a Sorginak at the service of Mari, Helena, a midwife that has the Control Ferility virtue and Metillia, a witch that enthrall her targets with her beautiful voice.

### CODE REVIEW

File: Magi/Brunhilde of the Black Forest/Brunhilde, the Corpse Witch.md [hunk 1]
```diff
@@ -33,21 +30,20 @@ Brunhilde is a witch living in a cavern, deep inside the Black Forest. She somet
 **Reputations:** The Bone Witch 1 (among the peasants)
 
 **Combat:**  
-*Dodge*: Init -1, Attack +0, Defense -1, Damage +0                                                                                                    
+*Dodge*: Init -1, Attack +0, Defense -1, Damage +0
```
Comment (GuillaumeDIDIER) [inline Magi/Brunhilde of the Black Forest/Brunhilde, the Corpse Witch.md]: There seems to be some trailing whitespace here ?

File: Magi/Helena of the Ostara Coven/Helena, the Midwife.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+**Characteristics:** Int +2, Per +0, Pre +2, Com +1, Str +0, Sta +0, Dex +1, Qik -1  
+**Size:** 0  
+**Age:** 22  
+**Decrepitude:** 0 (0)  
+**Warping Score:** 1 (8)  
+**Confidence Score:** 1 (3)  
+## Virtues  
+- **The Gift** _(special)_  
+- **Folk Witch** _(free, social status)_ cf. HMRE p34
+- **Control Fertility** _(major, supernatural)_ cf. HoH:Soc p106
```
Comment (GuillaumeDIDIER) [inline Magi/Helena of the Ostara Coven/Helena, the Midwife.md]: Isn't the shorthand HoH:S ?

File: Magi/Milia of the North Coven/Milia, the Animal Speaker.md [hunk 3]
```diff
@@ -41,35 +37,33 @@ Under the teachings of the witch, Milia refined her ability to summon animals an
...
 + Folk Witch Magic Theory 3 (Healing)  
-+ Folk Witches Lore 2 (rituals)  
 + Guile 1 (peasants)  
 + Healing 2 (wounds)  
 + Herbalism 3 (theriacs)  
-+ Hex 2 (face to face)  
 + Hunt 1 (traces)  
--  Language (native): South Slavonic 5 (Serbian)
-+ Magic Lore 2 (magical animals)  
+-  Language (native): South Slavonic 5 (Serbian)
```
Comment (GuillaumeDIDIER) [inline Magi/Milia of the North Coven/Milia, the Animal Speaker.md]: Shouldn't this line start with `+` instead of `-` ?

**Review by GuillaumeDIDIER — APPROVED** [review]
It seems mostly good, a few typos and good to go.

File: Magi/Brunhilde of the Black Forest/Brunhilde, the Corpse Witch.md (hunk 1 above)
Comment (interfector) [reply Magi/Brunhilde of the Black Forest/Brunhilde, the Corpse Witch.md]: Yes, I'll remove them.

File: Magi/Helena of the Ostara Coven/Helena, the Midwife.md (hunk 2 above)
Comment (interfector) [reply Magi/Helena of the Ostara Coven/Helena, the Midwife.md]: Yes, definitely.

File: Magi/Milia of the North Coven/Milia, the Animal Speaker.md (hunk 3 above)
Comment (interfector) [reply Magi/Milia of the North Coven/Milia, the Animal Speaker.md]: Probably, the rendered markdown it's the same but at least it'll be uniform.

**Comment by interfector** [comment]
I've commited a version with fixed typos. I should probably also look at other sheets because the trailing whitespaces are probably generated by my sheet generator :(

File: Magi/Milia of the North Coven/Milia, the Animal Speaker.md (hunk 3 above)
Comment (GuillaumeDIDIER) [reply Magi/Milia of the North Coven/Milia, the Animal Speaker.md]: It looks like you changed the line below (making it two lines in a row with a mismatched bullet point).

Comment (interfector) [reply Magi/Milia of the North Coven/Milia, the Animal Speaker.md]: > It looks like you changed the line below (making it two lines in a row with a mismatched bullet point).

I should have corrected the error, there might other lines like these in other character's sheets however.

**Pull request merged**